fiftyone/operators/builtins/panels/model_evaluation/__init__.py

```
@@ -0,0 +1,2620 @@
+            category=Categories.ANALYZE,
+            beta=True,
+            is_new=is_new("2024-11-07"),
+        )
+
+    def get_store(self, ctx):
+        return ctx.create_store(STORE_NAME)
+
+    def get_evaluation_id(self, dataset, eval_key):
+        try:
+            return str(dataset._doc.evaluations[eval_key].id)
+        except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)